modules/migration-migrating-on-prem-to-cloud.adoc

```diff
@@ -0,0 +1,114 @@
+----
+$ podman cp $(podman create registry.redhat.io/rhmtc/openshift-migration-controller-rhel8:v1.7.0):/crane ./
+----
+
+. Log in remotely to a node on the source cluster and a node on the destination cluster.
+
+. Obtain the cluster context for both clusters after logging in:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc config view
+----
+
+. Establish a tunnel by entering the following command on the command system:
++
+[source,terminal,sub="+quotes"]
+----
+$ crane tunnel-api [--namespace _<namespace>_] \
+      --destination-context _<destination-cluster>_ \
+      --source-context _<source-cluster>_
+----
++
+If you don't specify a namespace, the command uses the default value `openvpn`.
++
+For example:
++
+[source,terminal,subs="+quotes"]
+----
+$ crane tunnel-api --namespace my_tunnel \
+      --destination-context openshift-migration/c131-e-us-east-containers-cloud-ibm-com/admin \
+      --source-context default/192-168-122-171-nip-io:8443/admin
+----
++
+The command generates SSL Certificates. This process might take several minutes. A message appears when the process completes.
++
+The OpenVPN server starts on the destination cluster and the OpenVPN client starts on the source cluster.
++
+After a few minutes, the load balancer resolves on the source node.
++
+[TIP]
+====
+You can view the log for the OpenVPN pods to check the status of this process by entering the following commands with root privileges:
+
+[source,terminal,subs="+quotes"]
+----
+# oc get po -n _<namespace>_
+NAME            READY     STATUS      RESTARTS    AGE
+_<pod_name>_    2/2       Running     0           44s
+# oc logs -f -n _<namespace>_ _<pod_name>_ -c openvpn
+----
+When the address of the load balancer is resolved, the message `Initialization Sequence Completed` appears at the end of the log.
+====
+
+. On the openvpn server, which is on a destination control node, verify that the `openvpn` service and the `proxied-cluster` service are running:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc get service -n _<namespace>_
+----
+
+. On the source node, get the SA token for the migration controller:
++
+[source,terminal]
+----
+# oc sa get-token -n openshift-migration migration-controller
+----
+
+. Launch the {mtc-short} web console and add the source cluster, using the following values:
++
+* *Cluster name*: The source cluster name.
+* *URL*: `proxied-cluster._<namespace>_.svc.cluster.local:8443`. If you did not define a value for `_<namespace>_`, use `openvpn`.
+* *Service account token*: The token of the migration controller service account.
+* *Exposed route host to image registry*: `proxied-cluster._<namespace>_.svc.cluster.local:5000`. If you did not define a value for `_<namespace>_`, use `openvpn`.
+
+After {mtc-short} has successfully validated the connection, you can proceed to create and run a migration plan. The namespace for the source cluster should appear in the list of namespaces.
+
+
+.Additional resources
```

Comment:
This is duplicated in the assembly file. Please see that note for a comment on replacing the example `--help` command as a `[TIP]` admonition in this procedure module.

Reply:
Fixed.